Classes/Event/ProcessOrderCheckStockEvent.php

```diff
@@ -12,9 +12,17 @@
  */
 
 use Extcode\Cart\Domain\Model\Cart\Cart;
+use TYPO3\CMS\Core\Messaging\FlashMessage;
 
 final class ProcessOrderCheckStockEvent implements ProcessOrderCheckStockEventInterface
 {
+    private bool $allProductsAvailable = true;
+
+    /**
+     * @var FlashMessage[]
+     */
+    protected array $messages = [];
```

Comment:
Should we be more precise at this point? The messages are only output and processed further in the event of an error.
The method names should then also be adapted.

Reply:
Do you mean it should be called `$errorMessages`? Or what do you mean with more precise?

Reply:
This was my intention.